### PR TITLE
Webpage in master 1

### PR DESCRIPTION
**Pull request by stereobooster**

### CODE REVIEW

**Comment by stereobooster** [comment]
This is good to merge. What is inside:
- moved webpage to master
- deployment with travis-ci
- some refactoring

File: webpage/scripts/generate_readme.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env node
```
Comment (MicheleBertoli) [inline webpage/scripts/generate_readme.js]: Should we remove this file from the current PR?

File: webpage/src/components/Table.js [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+  } else if (value === false) {
+    return "";
+  } else {
+    return "?";
+  }
+};
+
+const CellTd = options => {
+  const { header, value } = options;
+  if (header === "Package") {
```
Comment (MicheleBertoli) [inline webpage/src/components/Table.js]: This recurring pattern (if Package else if Version) doesn't seem right.
Do you mind implementing another solution?
Like using a map, or different components.

File: webpage/src/components/Table.js [hunk 3]
```diff
@@ -9,6 +9,16 @@ const join = (array, separator) =>
     }
   }, []);
 
+const symbol = value => {
+  if (value === true) {
```
Comment (MicheleBertoli) [inline webpage/src/components/Table.js]: What are the possible values of `value` here?

File: webpage/scripts/generate_readme.js (hunk 1 above)
Comment (stereobooster) [reply webpage/scripts/generate_readme.js]: I will remove last commit from this PR

File: webpage/src/components/Table.js [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+  array.reduce((result, child, index) => {
+    if (index < array.length - 1) {
+      return result.concat(child, separator);
+    } else {
+      return result.concat(child);
+    }
+  }, []);
+
+const symbol = value => {
+  if (value === true) {
```
[the remark this answers is not in the document]
Comment (stereobooster) [reply webpage/src/components/Table.js]: `true`, `false`, `undefined`

File: webpage/src/components/Table.js (hunk 2 above)
Comment (stereobooster) [reply webpage/src/components/Table.js]:
```js
const fields = {
    "Package": value => { ... return join(links, " + "); },
    "Version": x => x,
    "default": symbol
}
const field = fields[header] || fields["default"]
return <td>{ field(value) }</td>
```

**Comment by stereobooster** [comment]
@MicheleBertoli done

**Comment by MicheleBertoli** [comment]
LGTM, thank you very much @stereobooster.

**Comment by MicheleBertoli** [comment]
Yay https://github.com/MicheleBertoli/css-in-js/commit/0e680fe2e984fdb9ec8e285c34a8dc6eb7592b7e
FYI, I had to pass the [no-sandbox](https://github.com/MicheleBertoli/css-in-js/commit/b05df8b890b121749037c50b89a1121a6884ecaa) option to puppeter.
Thank you very much one more time.

**Comment by stereobooster** [comment]
Probably missed it when was moving a website from one branch to another.

**Pull request merged**